tokio-trace-proc-macros/Cargo.toml

```diff
@@ -8,9 +8,10 @@ proc-macro = true
 
 [dependencies]
 tokio-trace = "0.1.0"
-syn = { version = "0.15", features = ["full", "extra-traits"] }
+syn = { version = "0.15", features = ["derive"] }
```

Comment:
I'm not sure this is doing what  I want it to do. The only part of syn that I think we need is `parse_macro_input!`. Running clean builds it still takes quite a few seconds to build `syn`.